### PR TITLE
update cloundfdry download location

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

GitHub action is failing because of Cloudfoundry

### Changes proposed in this pull request

updated the github location used for downloading
Cloundfoundry plugins

### Guidance to review

Check url

### Checklist

- [] Make sure all information from the Trello card is in here
- [] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: .github/workflows/deploy-paas.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Deploy publish app to PAAS
 
 on:
   push:
-    branches:
-      - master
+  #  branches:
```
Comment (defong) [inline .github/workflows/deploy-paas.yml:5]: Do we need this?

Comment (felixtheflex) [reply .github/workflows/deploy-paas.yml:5]: happy to remove. i.e. run on push; regardless of branch. it was the way i cloned it.

**Review by saliceti — APPROVED** [review]
Please make sure to put back the push: branches: master

File: .github/workflows/deploy-paas.yml (hunk 1 above)
Comment (felixtheflex) [reply .github/workflows/deploy-paas.yml]: > 
> 
> Please make sure to put back the push: branches: master

done

**Review by vigneshmsft — APPROVED** [review]

**Pull request merged**